examples/viewer.cpp

```diff
@@ -143,17 +158,24 @@ bool Viewer::render()
     {
         libfreenect2::Frame* frame = iter->second;
 
-        glViewport(x, y, width, height);
-        x += width;
-        if (x >= 1024)
+        glViewport(x, y, win_width, win_height);
```

Comment:
The bug is here. `win_width` is used for the width of the total window. There are 4 viewports in the windows, each of which should be half of the size. `width_width/2` here and so on for the rest of the code.